src/generic/datavgen.cpp

```diff
@@ -631,6 +633,101 @@ class wxDataViewTreeNode
 
 WX_DEFINE_SORTED_ARRAY_SIZE_T(unsigned int, wxDataViewSelection);
 
+
+struct RangeEntity
+{
+    unsigned int from;
+    unsigned int to;
```

Comment:
It would be nice to have a comment saying if this is inclusive or exclusive.

P.S. After reading the rest of the code, I see it's inclusive, but I also think that it would simplify the code a bit if it were exclusive (and it would definitely be more standard for C++ where intervals are always half-open).